app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -291,14 +295,14 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
                 className="flex flex-wrap w-[90%] gap-4 overflow-x-scroll justify-center pb-10 items-center"
                 data-testid="user-pick-history"
               >
-                {pickHistory?.map((logoURL, index) => {
+                {pickHistory?.map((team, index) => {
```

Comment:
Why do we need to refactor this code for this ticket?

Reply:
@shashilo 
The `alt` text for each logo used to be hardcoded as "team logo". We needed a way to assign it a dynamic value, ie the team's name.